homeassistant/components/climate/nest.py

```diff
@@ -29,8 +30,19 @@
         vol.All(vol.Coerce(int), vol.Range(min=1)),
 })
 
+TEMPERATURE_SCALE_SCHEMA = vol.Schema({
+    vol.Required(CONF_TEMPERATURE_UNIT): vol.In(
```

Comment:
This is not configuration so please use an `ATTR_` prefixed constant (you can define it locally if no suitable one already exists).